### PR TITLE
Clarify steps in managing open pull requests

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/developer/contribution-guide.md [hunk 1]
```diff
@@ -53,16 +53,23 @@ After submitting a PR, before it is merged:
...
     - If the build fails, check the error log to narrow down the reason  
     - Sometimes one of the multiple build tests will randomly fail due to issues in Travis CI. If you see just one build failure and no clear error message, it may be a random issue. Add a comment so the reviewer for your change can re-run the build for you, or close the PR and re-submit and that typically clears the issue. 
-2. Pull requests require review by an approved reviewer and an approved seconder.
-3. Feedback from reviewers needs to be addressed 
-3. If user interface changes do not match specification in Jira ticket, PM approval is needed.
-
-If you've included your mailing address in Step 1, you may receive a [Limited Edition Mattermost Mug](https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged. 
+2. PM review must pass
+    - A Product Manager will review the pull request to make sure it:
+        - fits with our product roadmap
```
Comment (lfbrock) [inline source/developer/contribution-guide.md]: Can you capitalize these bullet points?

File: source/developer/contribution-guide.md [hunk 2]
```diff
@@ -53,16 +53,23 @@ After submitting a PR, before it is merged:
...
+        - works as expected
+        - meets [UX guidelines](https://docs.mattermost.com/developer/fx-guidelines.html)
+    - The Product Manager may come back with some bugs or UI improvements to fix before the pull request moves on to dev review.
```
Comment (lfbrock) [inline source/developer/contribution-guide.md]: Can you update the bullet points to all be consistent with period or no period?

File: source/developer/contribution-guide.md [hunk 3]
```diff
@@ -53,16 +53,23 @@ After submitting a PR, before it is merged:
...
+    - Any comments will need to be addressed before the pull request is ready to merge
+
+If you've included your mailing address in the signed [Contributor License Agreement](https://www.mattermost.org/mattermost-contributor-agreement/), you may receive a [Limited Edition Mattermost Mug](https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged. 
 
 ### Approved Reviewers
 
 _Please DO NOT @-mention reviewers if they haven't yet commented on your PR._
 _Pull requests are reviewed in sequence._
 
 **Approved reviewers include**: coreyhulen, crspeller, hmhealey, jwilander
```
Comment (lfbrock) [inline source/developer/contribution-guide.md]: We should also update these lists to add George + Elias

Comment (jasonblais) [inline source/developer/contribution-guide.md]: Right, I meant to add them... thanks for the notice

**Comment by lfbrock** [comment]
+1

I think we're still a little inconsistent in this doc with combining fragments and sentences in the same list, but not sure if there's an easy way to make it better.

File: source/developer/contribution-guide.md [hunk 4]
```diff
@@ -42,27 +42,34 @@ Before submitting a pull request (PR), check that:
...
+        - Works as expected
+        - Meets [UX guidelines](https://docs.mattermost.com/developer/fx-guidelines.html)
+    - The Product Manager may come back with some bugs or UI improvements to fix before the pull request moves on to dev review.
+3. Dev review must pass
+    - Two developers will review the pull request and either give feedback or approve the PR.
+    - Any comments will need to be addressed before the pull request is ready to merge.
+
+If you've included your mailing address in the signed [Contributor License Agreement](https://www.mattermost.org/mattermost-contributor-agreement/), you may receive a [Limited Edition Mattermost Mug](https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged. 
 
 ### Approved Reviewers
```
Comment (esethna) [inline source/developer/contribution-guide.md]: We should probably split this out into approved dev and approved PM reviewers.

File: source/developer/contribution-guide.md [hunk 5]
```diff
@@ -42,27 +42,34 @@ Before submitting a pull request (PR), check that:
...
 
 _Please DO NOT @-mention reviewers if they haven't yet commented on your PR._
```
Comment (esethna) [inline source/developer/contribution-guide.md]: This porbably doesn't need to be italicized, it looks kinda funky

**Comment by jasonblais** [comment]
1. Removed the "do not @mention" note as I feel that's unnecessary.
2. Added approved PM reviewers
3. Removed seconders, which is no longer applicable for us

**Pull request merged**